pkg/dbmate/sqlite.go

```diff
@@ -165,6 +165,18 @@ func (drv SQLiteDriver) DeleteMigration(db Transaction, version string) error {
 	return err
 }
 
+// Does nothing currently
+func (drv SQLiteDriver) AcquireChangeLock(db *sql.DB) (bool, error) {
+	// FIXME use change locking for this driver also?
+	fmt.Println("Skipping migration locking because database driver is SQLite")
```

Comment:
I haven't done much research on this, but it looks like sqlite has something called an exclusive lock for the file - I wonder if we could use that.